sklearn/decomposition/fastica_.py

```diff
@@ -563,8 +565,7 @@ def inverse_transform(self, X, copy=True):
         """
         check_is_fitted(self, 'mixing_')
 
-        if copy:
```

Comment:
Should we set `copy=copy and self.whiten`? There is no need to trigger a copy is `self.whiten` is False.